Resources/Prototypes/_RMC14/Entities/Objects/Consumables/Drinks/cups.yml

```diff
@@ -62,9 +62,16 @@
         sprite: _RMC14/Objects/Consumable/Drinks/coffee_cup.rsi
         state: icon-3
         visible: false
+  - type: SolutionContainerManager
+    solutions:
+      drink:
+        maxVol: 30
```

Comment:
This is redundant, but if mugs are supposed to hold 30 units, just remove this and change it on the RMCCoffeeCup